OPAL/ProjectDependencies.mmd

```diff
@@ -1,31 +1,30 @@
 %%{ init: { 'flowchart': { 'curve': 'linear' } } }%%
```

Comment:
Should we remove the linear style here since it doesn't seem to be working anymore?

Reply:
Well it is the correct way to specify linear curves, and it only [because of a bug](https://github.com/mermaid-js/mermaid/releases/tag/mermaid%4011.6.0) that it is not working with the current latest version of `mermaid-cli`. I think we should leave it in and hope that the bugfix is incorporated into `mermaid-cli` soon, then the style specification should work again.